nmdc_schema/migrators/partials/migrator_from_10_2_0_to_11_0_0/migrator_from_10_2_0_to_11_0_0_part_17.py

```diff
@@ -0,0 +1,50 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+
+
+class Migrator(MigratorBase):
+    r"""
+    Migrates a database between two schemas.
+    """
+
+    _from_version = "XX"
+    _to_version = "PR250"
```

Comment:
I'll revisit this line of code. In (at least "some," but I think "all") other Berkeley migrators, we have been setting this to the PR identifier where the associated _schema change_ was made (as opposed to where the migrator was introduced). Since this is a bugfix, doing that (or interpreting it as a reader) might not be as straightforward as usual.

Reply:
I suggest:
1. Create a GitHub issue reporting the original bug
2. Set this `_to_version` value to `FIX_ISSUE_{number}`

I just made up that naming convention here. These values are not "interpreted" by any\* computer programs—At most, they are displayed verbatim in some logs.

_\*That I'm aware of or have been involved with_

Reply:
I'll do step 1 in about 15 minutes (if not already done)

Reply:
Oh, @kheal already did this. Here's the issue: https://github.com/microbiomedata/nmdc-schema/issues/2180

Here's my suggestion:
```suggestion
    _to_version = "FIXES_ISSUE_2180"
```